dom.go

```diff
@@ -404,6 +404,8 @@ func wrapHTMLElement(o *js.Object) HTMLElement {
 		return &HTMLUnknownElement{BasicHTMLElement: el}
 	case js.Global.Get("HTMLVideoElement"):
 		return &HTMLVideoElement{HTMLMediaElement: &HTMLMediaElement{BasicHTMLElement: el}}
+	case js.Global.Get("HTMLTemplateElement"):
+		return &HTMLTemplateElement{el}
```

Comment:
Please use a keyed field in this composite literal, for consistency with other elements.

```suggestion
		return &HTMLTemplateElement{BasicHTMLElement: el}
```